dask_sql/context.py

```diff
@@ -34,6 +34,12 @@
 from dask_sql.physical.rex import RexConverter, core
 from dask_sql.utils import ParsingException
 
+try:
+    import dask_cuda  # noqa: F401
+except ImportError:  # pragma: no cover
+    pass
+
+
```

Comment:
These need to be moved prior to line 12 (the first dask_sql import) which ends up starting the jvm.

Reply:
yep, good catch. I got in a hurry and forgot I had changed the ordering for testing.